test/blackbox-tests/test-cases/coq/compose-installed-compat.t/run.t

```diff
@@ -36,10 +36,6 @@ so this also tests that it won't be a problem.
   (warnings (deprecated_coq_lang_lt_08 disabled))
   Leaving directory 'B'
   $ dune install --root B --prefix=$PWD --display=short
-  Warning: Coq Language Versions lower than 0.8 have been deprecated in Dune
```

Comment:
Note that the warning still appears when building. This PR makes installation a little faster by making the dune file loading lazier.

cc @ejgallego just so that you're aware.